app/watchers/providers/docker/Docker.ts

```diff
@@ -366,32 +365,29 @@ class Docker extends Component {
                 );
                 this.log.debug(err);
             } else {
-                stream.on('data', (chunk) => this.onDockerEvent(chunk));
+                stream!.on('data', (chunk) => this.onDockerEvent(chunk));
             }
         });
     }
 
     /**
      * Process a docker event.
-     * @param dockerEventChunk
-     * @return {Promise<void>}
      */
-    async onDockerEvent(dockerEventChunk) {
+    async onDockerEvent(dockerEventChunk: string) {
         const dockerEvent = JSON.parse(dockerEventChunk.toString());
         const action = dockerEvent.Action;
         const containerId = dockerEvent.id;
 
         // If the container was created or destroyed => perform a watch
         if (action === 'destroy' || action === 'create') {
-            await this.watchCronDebounced();
+            this.watchCronDebounced!();
```

Comment:
I think I broke something, just not sure yet how exactly, the function can be null right now.
```suggestion
            this.watchCronDebounced?.();
```